pkg/kubelet/prober/prober_manager.go

```diff
@@ -222,6 +246,18 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *v1.PodStatus) {
 			ready = !exists
 		}
 		podStatus.ContainerStatuses[i].Ready = ready
+
```

Comment:
Add comment about why this is safe when the startup feature is disabled, e.g. `started  ==  !running` when startup probe is disabled.

Reply:
And do we want to always set `started = true` when the feature is disabled? That seems safer to me. The current PR still changes some workflow when the feature is disabled.

Reply:
Ok, so you want me to add the feature flag here too, and set `started = true` in that case?

Reply:
@matthyx That seems safer to me. WDYT?